test/Address.test.js

```diff
@@ -0,0 +1,22 @@
+const AddressImpl = artifacts.require('AddressImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('chai')
+  .should();
+
+contract('Address', function ([_, creator]) {
+  beforeEach(async function () {
+    this.mock = await AddressImpl.new();
+  });
+
+  it('should return false for account address', async function () {
+    const isContract = await this.mock.isContract(creator);
```

Comment:
Could you replace this (and the other test) for `(await this.mock.isContract(creator)).should.equal(false);`? See #1216